charts/ceph-csi-cephfs/templates/provisioner-rules-clusterrole.yaml

```diff
@@ -37,4 +37,11 @@ rules:
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch", "update", "patch"]
 {{- end -}}
+{{- if semverCompare ">=1.14" .Capabilities.KubeVersion.GitVersion }}
+{{- if .Values.registerDriver }}
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["csidrivers"]
+    verbs: ["create", "delete"]
```

Comment:
I guess `update` is not needed yet, but maybe a future version of the CSIDriver object needs to adjust its registration?